analysis_notebooks/Attribute Analysis.ipynb

```diff
@@ -1237,7 +1671,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Ratio of average distance between attribute and object. Negative values represent that the attribute is **closer** to the object, while positive values represent that the attribute is **further** from the object (compared to all other attributes)."
+    "Ratio of average distance between attribute and object. Negative values represent that the attribute is **closer** to the object, while positive values represent that the attribute is **further** from the object (compared to all other attributes).\n",
+    "\n",
+    "Ordinal graph displays analysis of distance as a function of some ordinal variable. The graph displays the **mean** and **median** distance of an object from a person by attribute. Sample images from the specified percentile (of all distances between person and attribute) in each attribute are displayed. "
```

Comment:
is there a way to display this markdown only if ordinal attributes are even relevant here? else to not display this